themes/socialblue/templates/node/event/node--event--full--sky.html.twig

```diff
@@ -0,0 +1 @@
+{% extends "node--full--sky.html.twig" %}
```

Comment:
In this case, this is necessary, since we already have a separate template `node--event--full`, and in sky theme, full template is simpler. so we can use only one.